csharp/ql/test/resources/stubs/_frameworks/Microsoft.NETCore.App/System.Runtime.InteropServices.cs

```diff
@@ -1974,155 +1290,620 @@ public struct ManagedToUnmanagedIn
+                static bool System.Numerics.INumberBase<System.Runtime.InteropServices.NFloat>.TryParse(System.ReadOnlySpan<char> s, System.Globalization.NumberStyles style, System.IFormatProvider provider, out System.Runtime.InteropServices.NFloat result) => throw null;
+                static bool System.ISpanParsable<System.Runtime.InteropServices.NFloat>.TryParse(System.ReadOnlySpan<char> s, System.IFormatProvider provider, out System.Runtime.InteropServices.NFloat result) => throw null;
+                public static bool TryParse(System.ReadOnlySpan<char> s, out System.Runtime.InteropServices.NFloat result) => throw null;
+                static bool System.Numerics.INumberBase<System.Runtime.InteropServices.NFloat>.TryParse(string s, System.Globalization.NumberStyles style, System.IFormatProvider provider, out System.Runtime.InteropServices.NFloat result) => throw null;
+                static bool System.IParsable<System.Runtime.InteropServices.NFloat>.TryParse(string s, System.IFormatProvider provider, out System.Runtime.InteropServices.NFloat result) => throw null;
+                public static bool TryParse(string s, out System.Runtime.InteropServices.NFloat result) => throw null;
+                bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteExponentBigEndian(System.Span<byte> destination, out int bytesWritten) => throw null;
+                bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteExponentLittleEndian(System.Span<byte> destination, out int bytesWritten) => throw null;
+                bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteSignificandBigEndian(System.Span<byte> destination, out int bytesWritten) => throw null;
+                bool System.Numerics.IFloatingPoint<System.Runtime.InteropServices.NFloat>.TryWriteSignificandLittleEndian(System.Span<byte> destination, out int bytesWritten) => throw null;
+                public double Value { get => throw null; }
+                static System.Runtime.InteropServices.NFloat System.Numerics.INumberBase<System.Runtime.InteropServices.NFloat>.Zero { get => throw null; }
             }
             namespace ObjectiveC
             {
                 public static class ObjectiveCMarshal
                 {
-                    public enum MessageSendFunction : int
+                    public static System.Runtime.InteropServices.GCHandle CreateReferenceTrackingHandle(object obj, out System.Span<nint> taggedMemory) => throw null;
+                    public static unsafe void Initialize(delegate* unmanaged<void> beginEndCallback, delegate* unmanaged<nint, int> isReferencedCallback, delegate* unmanaged<nint, void> trackedObjectEnteredFinalization, System.Runtime.InteropServices.ObjectiveC.ObjectiveCMarshal.UnhandledExceptionPropagationHandler unhandledExceptionPropagationHandler) => throw null;
```

Comment:
Is `nint` correct here? It was `System.IntPtr` before. (And it shows up as `System.IntPtr` in the docs)

Reply:
I believe `nint` and `System.IntPtr` is the same thing.